instrumentation/kamon-armeria/src/main/scala/kamon/armeria/instrumentation/converters/FutureConverters.scala

```diff
@@ -0,0 +1,19 @@
+package kamon.armeria.instrumentation.converters
+
+import java.util.concurrent.{Future => JFuture}
+
+import scala.concurrent.{ExecutionContext, Future, Promise}
+import scala.util.Try
+
+trait FutureConverters {
+  implicit class FutureConverterOps[T](jFuture: JFuture[T])(implicit ec: ExecutionContext) {
+    def toScala: Future[T] = {
+      val promise = Promise[T]()
+      Future {
+        promise.complete(Try(jFuture.get))
```

Comment:
`jFuture.get()` is a blocking IO. It will block Armeria event loop. Armenia always return [CompletableFuture](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html) for asynchronous functions. CompletableFuture is compatible that means you can convert it to Scala Future. 
scala-java8-compat could be a good reference. 😀
https://github.com/scala/scala-java8-compat/blob/8123acd42f5646e41f4d13329fc2ef8f554cc63c/src/main/scala/scala/compat/java8/FutureConverters.scala#L88-L92

Reply:
Thanks for the review, i'll work on it 😄